package.json

```diff
@@ -2,7 +2,7 @@
   "name": "smooth-gradient-sass-function",
   "version": "2.1.0",
   "description": "You can get smooth gradient color-stops map for sass linear-gradient",
-  "main": "_gradients.scss",
+  "main": "_index.scss",
```

Comment:
`main` field value required js/cjs/mjs file.

Reply:
```suggestion
```

Not required, so we can just remove since there's no JS to `require`. I've been putting the sass entrypoint there as my own convention but yeah, not correct.